src/applications/vaos/appointment-list/redux/actions.js

```diff
@@ -168,11 +168,79 @@ function getAdditionalFacilityInfoV2(appointments) {
     .filter(n => n);
 }
 
+export function fetchPendingAppointments() {
```

Comment:
Moved function before 'fetchFutureAppointments' to remove eslint warning: no-use-before-define